cli/azd/internal/scaffold/spec.go

```diff
@@ -98,27 +98,56 @@ type ServiceSpec struct {
 	Name string
 	Port int
 
-	Env map[string]string
```

Comment:
Seems this environment variable is not used in the bicep template before this PR, seems it's a unfinished feature.